### DISCUSSION TITLE
A bit of help :)

### DISCUSSION THREAD

**Discussion by JinParkmida** [marco97pa/Blackpink-Data#29] [General]
Hi there, 
First of all I really love your project idea. 
It seems like I am too stupid to figure it out on my own, would you kindly help me on how to add more groups to the list? :)

**Comment by marco97pa** [comment]
Hi, thanks for your interest, I really appreciate it.
The code you see in this repository is actually running on the cloud every 15 minutes and every new information about Blackpink is being posted on the Twitter account @data_blackpink.
So adding a new group or artist to this repository is not a good idea since it is focused on BLACKPINK.
But you can actually fork this code, creating your own repository and change the data.yaml file according to your group.
By editing this file you can change the group name, the members, the relevant social media accounts to check and more.
You can read more information about how to change the data.yaml file in the documentation and in the README.md
